lib/extras.js

```diff
@@ -47,15 +47,15 @@ export async function extendRoutes(routes, options) {
 function mapSimpleAliases(route, aliases) {
   return aliases.map((alias, index) => Object.assign({}, route, {
     path: alias,
-    name: `${route.name}-p${alias}`
+    name: `${route.name}${alias}`
```

Comment:
I think it would be better if we set name to `route.name` and user `alias` for fallback.

Reply:
This is what happens for main (non aliased) routes. Alias is set to an empty string. For aliases, the separator is `/`

Reply:
Sure you're right